java/core/src/main/java/com/microsoft/bond/BlobBondType.java

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.bond;
+
+import java.io.IOException;
+
+/**
+ * Implements the {@link BondType} contract for the Bond "blob" data type.
+ */
+public final class BlobBondType extends PrimitiveBondType<byte[]> {
```

Comment:
Yes, I realized that yesterday and changed it already.

---
In reply to: [123117294](https://github.com/Microsoft/bond/pull/502#discussion_r123117294) [](ancestors = 123117294)